content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -561,6 +562,13 @@ Each feature gate is designed for enabling/disabling a specific feature:
   The Job controller uses Pod finalizers and a field in the Job status to keep
   track of the finished Pods to count towards completion.
 - `KMSv2`: Enables KMS v2 API for encryption at rest. See [Using a KMS Provider for data encryption](/docs/tasks/administer-cluster/kms-provider) for more details.
+- `KubeletCgroupDriverFromCRI`: Enable detection of the kubelet cgroup driver
+  configuration option from the CRI. The container runtime also needs to support
```

Comment:
I find the sentence 
```
The container runtime also needs to support this feature in which case the kubelet will ignore the cgroupDriver...
```

to be kind of clunky. Maybe 
```
This feature gate requires the user run a container runtime that supports the `RuntimeConfig` CRI call. If both CRI and Kubelet support this feature, the kubelet will ignore the cgroupDriver (--cgroup-driver) configuration optoin. ...

```

Reply:
@haircommander changed as suggested. I left the last sentence (`If the container runtime doesn't support it, the kubelet will fallback to using its cgroupDriver option.`) intact.